wings/market/binance_market.pyx

```diff
@@ -366,7 +367,7 @@ cdef class BinanceMarket(MarketBase):
 
     @property
     def name(self) -> str:
-        return "binance"
+        return "Binance"
 
```

Comment:
Is there a reason for us to change the Market names to uppercase? If not, it's better to keep it consistent since the CLI uses all lowercase